File: Baekjoon/정권호/Q2438.java
import java.io.BufferedReader;
import java.io.InputStreamReader;
import java.io.IOException;

class Main {
    public static void main(String[] args) throws IOException {
        BufferedReader bufferedReader = new BufferedReader(new InputStreamReader(System.in));
        int a = Integer.parseInt(bufferedReader.readLine());
        StringBuilder result = new StringBuilder();

        for (int i = 0; i < a; i++) {
            for (int j = 0; j <= i; j++) {
                result.append("*");
            }
            result.append("\n");
        }
        System.out.print(result);
    }
}

File: Baekjoon/정우빈/Q2438.kt
fun main() {
    val a = readLine()!!.toInt()
    val str = StringBuilder()
    val star = StringBuilder()

    for (i in 1..a) {
        star.append("*")
        str.append(star).append("\n")

//        str.append("*".repeat(i)).append("\n")
//        for(j in 1..i) {
//            print("*")
//        }
//        println()
    }
    println(str)
}